lms/static/sass/base/_variables.scss

```diff
@@ -187,6 +187,7 @@ $m-blue-t0: rgba($m-blue,0.125);
 $m-blue-t1: rgba($m-blue,0.25);
 $m-blue-t2: rgba($m-blue,0.50);
 $m-blue-t3: rgba($m-blue,0.75);
+$m-blue-hover: rgb(223,242,247); // #DFF2F7
```

Comment:
@marcotuts can you please suggest any existing colour that we can use for this hover state ?